atomata/capstone_WebClient
# Texteditor v1

### Summary 
This PR adds the text editor to save/modify text for each action in the action sequence
### Details
_Fixes the  WebgL input consumption bug
_ adds the styled text editor to the editor page 
_ allows saving and viewing the text for each action
### How to test?
_create new experience , add some texts to different actions in action sequence ( save the texts) , save the experience, then reload the experience and check if the texts are there for each action
### Checks
